pages/common/vagrant.md

```diff
@@ -0,0 +1,19 @@
+# vagrant
+
+> Configure lightweight, reproducible, and portable development environments
+
+- Create Vagrantfile in current folder with the given Vagrant box
+
+`vagrant init hashicorp/precise32`
```

Comment:
Yes, I get that, but to quote an earlier comment of yours, if we don't add that detail (that it is the default vagrant box) to the example descriptions, users "will have to go and read yet another documentation" to know that. I mean, I could figure out that precise32 means a 32-bit Ubuntu 12.04 Precise Pangolin, but what about hashicorp?

I think it's worth including that detail in the description to make it clear why we chose that specific example. So instead of "the given Vagrant box" it would be "the default Vagrant box". What do you say?